soccer/gameplay/plays/Defensive_Scramble/basic_defensive_scramble.py

```diff
@@ -0,0 +1,77 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+
+import standard_play
+import tactics.positions.submissive_goalie as submissive_goalie
+import tactics.positions.submissive_defender as submissive_defender
+import evaluation.opponent as eval_opp
+import tactics.positions.wing_defender as wing_defender
+import skills.mark as mark
+import skills.capture as capture
+import tactics.defense
+import situational_play_selection
+
+## Play that uses submissive defenders and wingers to defend
+#  an attack close to our goal and cause one robot to capture.
+#  
+#  By default, we will use standard defense (two submissive 
+#  defenders, one goalie) and two wing defense robots. The
+#  remaining robot will mark the highest threat robot. 
+class BasicDefensiveScramble(standard_play.StandardPlay):
```

Comment:
For what it's worth, I think the whole standard play thing has kinda blown up. It was just used cause we didn't want to add a defense to every play we make manually. If you want to control the defense, I think it would be better to pull from `play.Play` and do the correct overrides. @liquidmetal9015 will have to make some changes, but I think `play.Play` was supposed to be the base play class that everything inherits from and `standardPlay` was moreso for the lazy people who didn't want to add a defense and goalie every single time for things like adaptive, or basic122

Reply:
Thanks for letting me know. I think I got a bit weird in this situation, but I do want to use standard defense in this situation, so standard play is the way to go.